firefox_puppeteer/base.py

```diff
@@ -28,16 +28,44 @@ def get_marionette(self):
 class UIBaseLib(BaseLib):
     """A base class for all UI element wrapper classes inside a chrome window."""
 
-    def __init__(self, marionette_getter, window):
-        BaseLib.__init__(self, marionette_getter)
-
+    def __init__(self, marionette_getter, parent, element):
```

Comment:
My proposal is not to add "parent" to our classes at all, at least not at this time.